pkg/sysfs/syspath.go

```diff
@@ -86,7 +87,7 @@ func (s Device) getParent() (string, bool) {
 	// nvme namespace. Length checking is to avoid index out of range in case of malformed
 	// links (extremely rare case)
 	for i, part := range parts {
-		if part == NVMeSubSystem {
+		if part == NVMeSubSystem || part == altNVMeSubSystem {
```

Comment:
Add test cases for this change [here](https://github.com/openebs/node-disk-manager/blob/develop/pkg/sysfs/syspath_test.go#L29)

Reply:
Done.